rpc/documentation/api.md

```diff
@@ -377,6 +377,8 @@ The service provides the following methods:
 - [`TransactionNotifications`](#transactionnotifications)
 - [`SpentnessNotifications`](#spentnessnotifications)
 - [`AccountNotifications`](#accountnotifications)
+- [`StartTicketPurchase`](#startticketpurchase)
+- [`StopTicketPurchase`](#stopticketpurchase)
```

Comment:
wrong service, you added these to the WalletLoaderService not the WalletService.